tests/test_resolver.py

```diff
@@ -267,7 +267,7 @@ def test_run_steps_not_acceptable(
 
         resolver._init_context()
 
-        resolver.predictor.should_receive("set_reward_signal").with_args(math.nan).once()
+        resolver.predictor.should_receive("set_reward_signal").with_args(state1, math.nan).once()
```

Comment:
`state1`?

Reply:
Yes - https://github.com/thoth-station/adviser/blob/55dcadb2ca8d25dd4ba6864a36bad38b7577e9e0/tests/test_resolver.py#L250

Reply:
Thanks, just to be sure :)